corehq/apps/export/management/commands/prepare_multimedia_export.py

```diff
@@ -0,0 +1,112 @@
+from datetime import datetime
+import traceback
+from django.core.management.base import BaseCommand
+from corehq.apps.export.const import MAX_MULTIMEDIA_EXPORT_SIZE
+
+from corehq.apps.export.dbaccessors import get_properly_wrapped_export_instance
+from corehq.apps.export.export import get_export_query
+from corehq.apps.export.models.new import DatePeriod
+
+
+def _forms_with_attachments(es_query):
+    query = es_query.source(['_id', 'external_blobs'])
+
+    for form in query.scroll():
+        try:
+            for attachment in form.get('external_blobs', {}).values():
+                if attachment['content_type'] != "text/xml":
+                    yield form
+                    continue
+        except AttributeError:
+            pass
+
+
+def create_date_period(start_date, end_date):
+    start_date = datetime.strptime(start_date, '%d-%m-%Y')
+    end_date = datetime.strptime(end_date, '%d-%m-%Y')
+
+    if end_date.day is start_date.day:
+        days = 1
+    else:
+        days = end_date.day - start_date.day
+
+    return DatePeriod(
+        period_type="range",
+        days=days,
+        begin=start_date,
+        end=end_date,
+    )
+
+
+def convert_bytes(size):
+    for x in ['bytes', 'KB', 'MB', 'GB', 'TB']:
+        if size < 1024.0:
+            return "%3.1f %s" % (size, x)
+        size /= 1024.0
+
+    return size
+
+
+class Command(BaseCommand):
+    help = "Gets multimedia files linked to export instance id"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'export_id',
+            help="The id of the export which files are wanted"
+        )
+        parser.add_argument(
+            'user',
+            help="The id of the user to filter by"
+        )
+        parser.add_argument(
+            'start_date',
+            help="The start date to filter by"
+        )
+        parser.add_argument(
+            'end_date',
+            help="The end date to filter by"
+        )
```

Comment:
It might be nice to have reasonable defaults for these arguments, if possible. For example, using the last month if you haven't specified a date range, and using all users if you don't specify a user